Makefile

```diff
@@ -0,0 +1,21 @@
+PREFIX=/usr/
+DESTDIR=/
+BINARY_NAME=apx
+
+all: build
+
+build:
+	go build -o ${BINARY_NAME}
+
+install:
+	install -Dm755 ${BINARY_NAME} ${DESTDIR}${PREFIX}/bin/${BINARY_NAME}
+	mkdir -p ${DESTDIR}/etc/apx
+	sed -i 's|/usr/share/apx/distrobox|${PREFIX}/share/apx/distrobox|g' config/config.json
+	install -Dm644 config/config.json ${DESTDIR}/etc/apx/config.json
+	mkdir -p ${DESTDIR}${PREFIX}/share/apx
+	sh distrobox/install --prefix ${DESTDIR}${PREFIX}/share/apx
```

Comment:
is there a reason to change the current distrobox destination `/usr/lib/apx/` to `/usr/share/apx/`?

Reply:
`/usr/share/apx` seems more fitting since `/usr/lib` is meant for system libraries and to me distrobox isn't really a library for apx.
Although this change isn't very important, it could be reverted if needed.

Reply:
could /usr/local/share also be an option? (i'm pretty sure /usr/share is read only on silverblue/ostree)

Reply:
make allows changing these variables, I updated the readme to show how to change them.

Reply:
Oh okay, thanks